keyboards/al1/rules.mk

```diff
@@ -0,0 +1,30 @@
+MCU = atmega32u4
+
+F_CPU = 16000000
+
+ARCH = AVR8
+
+F_USB = $(F_CPU)
+
+OPT_DEFS += -DINTERRUPT_CONTROL_ENDPOINT
+
+OPT_DEFS += -DBOOTLOADER_SIZE=4096
```

Comment:
Do we know what controller is being used here? 
Eg, is this a Pro Micro, Teensy 2.0, other, or SMD?